tests/network/test_add_intercept.py

```diff
@@ -334,3 +335,91 @@ async def test_add_intercept_blocks(websocket, context_id, url_patterns,
         },
         "type": "event",
     }
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("url_patterns", [
+    [
+        {
+            "type": "string",
+            "pattern": "https://www.example.com/",
```

Comment:
Can you use fixture instead of `https://www.example.com/`?

Reply:
It's not possible to use fixtures in parameters: https://github.com/pytest-dev/pytest/issues/349

Reply:
We could expose a global string instead of a fixture. Or we could break the test down into multiple functions so that it is not parameterized anymore. Otherwise it's not really possible at the moment.

Reply:
Actually, I forgot to mention a third possibility: We could use a pytest plug-in that allows so. Do you have a preference? See the upstream issue above.